tests/dnfserver/container/nginx.conf

```diff
@@ -0,0 +1,70 @@
+server {
+    listen       8443 ssl;
+    listen       [::]:8443 ssl;
+    server_name  localhost;
+    root         /dnfrepo;
+
+    ssl_certificate "/etc/pki/nginx/server.crt";
+    ssl_certificate_key "/etc/pki/nginx/server.key";
+
+    # enable client cert verification
+    ssl_client_certificate /etc/pki/myCA.crt; ## Use your own trusted certificate from CA/SSLTrust
+    ssl_verify_client on;
+
+    ssl_session_cache shared:SSL:1m;
+    ssl_session_timeout  10m;
+    ssl_ciphers PROFILE=SYSTEM;
+    ssl_prefer_server_ciphers on;
+    ssl_verify_depth 2;
+
+    # if you need to debug
+    error_log /var/log/nginx/error.log debug;
+
+
+    error_page 404 /404.html;
+        location = /40x.html {
+    }
+
+    error_page 500 502 503 504 /50x.html;
+        location = /50x.html {
+    }
+}
+
+server {
+    listen       8082;
+    listen       [::]:8082;
+    server_name  localhost;
+    root         /certificates;
+
+    # if you need to debug
+    error_log /var/log/nginx/error.log debug;
+
+
+    error_page 404 /404.html;
+        location = /40x.html {
+    }
+
+    error_page 500 502 503 504 /50x.html;
+        location = /50x.html {
+    }
+}
+
+
+server {
+    listen       8081;
+    listen       [::]:8081;
+    server_name  localhost;
+    root         /dnfrepo;
+
+    # if you need to debug
+    error_log /var/log/nginx/error.log debug;
+
+
+    error_page 404 /404.html;
+        location = /40x.html {
+    }
+
+    error_page 500 502 503 504 /50x.html;
+        location = /50x.html {
```

Comment:
Is this one just for debugging without SSL?

Reply:
yes, exactly.